src/main/java/core/basesyntax/service/activitieswithfruitservice/ActivitiesStrategy.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service.activitieswithfruitservice;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface ActivitiesStrategy {
+    void performTransaction(String code, FruitTransaction fruitTransaction);
```

Comment:
```suggestion
public interface TransactionStrategy {
    TransactionHandler getTransactionHandler( FruitTransaction fruitTransaction);
```

Reply:
not fixed. you can't just ignore Mentor's suggestions. if you have any questions or need some help - they should be asked in the platform's chats